source/CLI.html.md

```diff
@@ -5,6 +5,8 @@ summary: 'plm is a little CLI tool that can help you to manage your instance fro
 command line if you are admin of an instance.'
 ---
 
+You can see these help messages with `plm --help` or `plm COMMAND --help` (where `COMMAND` is the specific subcommand you are interested in).
+
 If any required argument is ommitted, you will be asked to input manually.
 
 ## `plm instance`
```

Comment:
insert a line break here

different markdown engines will not treat this as header otherwise

Reply:
I'm not sure of where I should add the line break, there already are blank lines before and after the heading? (and both GitHub and Middleman's engines render it correctly apparently, so…)

Reply:
before and after, yes

also, sorry, this looked very different on my phone in the dark, to extremely tired eyes 😩